micro-rdk-server/esp32/main.rs

```diff
@@ -34,7 +34,12 @@ mod esp32 {
 
         micro_rdk::esp32::esp_idf_svc::log::EspLogger::initialize_default();
 
-        let repr = RobotRepresentation::WithRegistry(Box::<ComponentRegistry>::default());
+        let mut r = Box::<ComponentRegistry>::default();
+        if let Err(e) = micro_rdk_modular_driver_example::free_heap_sensor::register_models(&mut r)
```

Comment:
I think you should also register the RSSI sensor, or at least that is what the ticket seems to ask for. I think you will need to adjust the visibility of the register function.

Reply:
@EshaMaharishi is this in line with what you wanted for features included in the binary hosted on github, at the same time users will reimplement/re-register the WIFI sensor when learning to add modules?

Reply:
I wanted to leave at least one sensor that doesn't require any hardware that works with the modular driver setup instructions.

Though, now I'm confused again... did we conclude that if we add a sensor this way, it will only affect the micro-rdk-installer flow, not the cargo flow?